### ISSUE TITLE
Error: Cannot find module '../../package.json'

### ISSUE THREAD

**Issue by ledsun** [raineorshine/npm-check-updates#990]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 10.17`

I'm using the ncu 12.0.1 and Node.js 17.0.1 and 16.13.0 both. 

---------------------------

## Steps to Reproduce

```sh
ncu
```

## Current Behavior

```sh
ledsun@MSI:~/textae►ncu
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '../../package.json'
Require stack:
- /home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js:11:40)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1147:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js'
  ]
}

Node.js v17.0.1
```

## Expected Behavior

```sh
ledsun@MSI:~/textae►ncu
Checking /home/ledsun/textae/package.json
[====================] 48/48 100%

 ajv                         8.6.3  →   8.7.1
 @typescript-eslint/parser   5.3.0  →   5.3.1
 eslint                     7.32.0  →   8.2.0
 terser-webpack-plugin      ^5.2.4  →  ^5.2.5
 webpack                    5.61.0  →  5.62.1

Run ncu -u to upgrade package.json
```

I changed `"../../package.json"` to `"../../../package.json"` in `/home/ledsun/.local/share/nvm/v17.0.1/lib/node_modules/npm-check-updates/build/src/bin/cli.js:11`, then the ncu runs expected.

**Comment by pfbrowning** [comment]
This affects me also on node 16.13.  The suggested workaround worked for me.

**Comment by virtualpatterns** [comment]
Ditto ... same issue and the workaround worked.

**Comment by jcurlier** [comment]
+1

**Comment by EdwardAngert** [comment]
another +1 for both the issue and the workaround
(and linking what I think is a dupe issue in #991 )

line 11 of `/usr/local/lib/node_modules/npm-check-updates/build/src/bin/cli.js`:

```
const package_json_1 = __importDefault(require("../../../package.json"));
```

**versions**

node:

```shell
$ node -v
v17.0.1
```

ncu:

```shell
$ ncu -V
12.0.1
```

**Comment by tluchowski** [comment]
+1

**Comment by rakeshramakrishnan-hbk** [comment]
+1

**Comment by k-frankov** [comment]
+1

**Comment by almothafar** [comment]
Same issue with latest node and npm, editing `npm-check-updates/build/src/bin/cli.js` workaround worked for me

**Comment by vitalii0508** [comment]
+1

**Comment by raineorshine** [comment]
Thanks for reporting and for providing a workaround!

> Ditto ... same issue and the workaround worked.

@virtualpatterns I see that you are validating the OP to add confidence for other users. I encourage you to use github's built-in reactions like 👍 rather than create a "ditto" or "same" comment, which adds noise to the discussion and increases notifications.

**Comment by raineorshine** [comment]
Fixed and published in `v12.0.2`.

**Comment by hassnaseer** [comment]
I had run ncu -u its update my package.json file but after i npm install it not working.
